### PR TITLE
Fix the scrollbar issue in widget container

### LINKED ISSUE #1001: sp_whoisactive and Server Reports insights extensions don't have vertical scroll
*Note: issue reported by @cirinovargas on gitter*
- SQL Operations Studio Version: 0.27.3

Steps to Reproduce:
1. Show insights page

Expected: Scrollbars if the content overflows

Actual: No scrollbars until window is resized.

image

### PR DESCRIPTION
**Pull request by AbbiePetcht**
fix #1001: sp_whoisactive and Server Reports insights extensions don't have vertical scroll

### CODE REVIEW

**Comment by anthonydresser** [comment]
Does this negatively affect other widgets? or anything else in the dashboard.

**Comment by anthonydresser** [comment]
What was the underlying issue here?

**Comment by AbbiePetcht** [comment]
@anthonydresser The underlying issue is that when dashboard extension renders off dom, it has an incorrect height. When you select on extension tab like "server reports" or "sp_whoisactive" the scrollbar won't show up. The fix is just calling the widgetContent layout before setting the scroll dimensions in layout method

**Comment by anthonydresser** [comment]
@AbbiePetcht but when it's put on dom (i.e navigated to) it should fix it's height and notify it's parents that it's height change, which should change the scroll dimensions.

**Comment by AbbiePetcht** [comment]
@anthonydresser I already moved scroll bar to widget content. Please take a look at my changes

File: src/sql/parts/dashboard/common/dashboardPage.component.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ export abstract class DashboardPage extends AngularDisposable {
 						configs = cb.apply(this, [configs, this.dashboardService, this.context]);
 					});
 					this._gridModifiers.forEach(cb => {
-						configs = cb.apply(this, [configs, this._originalConfig]);
```
Comment (anthonydresser) [inline src/sql/parts/dashboard/common/dashboardPage.component.ts]: does this brake editing?

Comment (AbbiePetcht) [reply src/sql/parts/dashboard/common/dashboardPage.component.ts]: The home tab doesn't call function. This function is for dashboard extension

---
In reply to: [178911833](https://github.com/Microsoft/sqlopsstudio/pull/1016#discussion_r178911833) [](ancestors = 178911833)

**Pull request merged**